tmp.py

```diff
@@ -0,0 +1,61 @@
+# %%
+from fixedpointmath import FixedPoint, FixedPointMath
```

Comment:
This doesn't look like it should be in the repo

Reply:
oops crap thanks